deploy/infra/lib/constructs/app-backend.ts

```diff
@@ -167,6 +166,18 @@ export class AppBackend extends Construct {
       })
     );
 
+    this._lambda.addToRolePolicy(
+      new PolicyStatement({
+        actions: ["sts:AssumeRole"],
+        resources: ["*"],
+        conditions: {
+          StringEquals: {
+            "iam:ResourceTag/CommonFate": "GrantedApprovalsSSOIdentityProvider",
```

Comment:
proposing a change here on the tag key so that it is clearer it is used for ABAC (otherwise, users might omit the tag when deploying the role if they think it is just for metadata)

```suggestion
            "iam:ResourceTag/assumed-by-common-fate-api": "true",
```